src/rprblender/engine/context_hybridpro.py

```diff
@@ -101,3 +101,7 @@ def sync_catchers(self, use_transparent_background=False):
     def create_tiled_image(self, key):
         # Tiled images are unsupported by HybridPro
         return None
+
+    def sync_auto_adapt_subdivision(self, width=0, height=0):
```

Comment:
Additional question to this: why KeyError happen here:
`obj.set_auto_adapt_subdivision_factor(fb, camera, obj.subdivision['factor'])`

Reply:
When you start Interactive Viewport render it works well after you make changes to any material with `cycles.displacement_method` `'DISPLACEMENT'` or `'BOTH'` during sync error occurs.

Reply:
Thanks for expalantion